tabbycat/draw/manager.py

```diff
@@ -184,17 +183,25 @@ def get_relevant_options(self):
     def get_teams(self):
         """Get teams in ranked order."""
         teams = super().get_teams()
-        if self.round.tournament.pref('draw_pullup_restriction') == 'least_to_date':
-            annotate_npullups(teams, self.round.prev)
 
         metrics = self.round.tournament.pref('team_standings_precedence')
-        generator = TeamStandingsGenerator(metrics, ('rank', 'subrank'), tiebreak="random")
+        extra_metrics = []
+
+        if self.round.tournament.pref('draw_pullup_restriction') == 'least_to_date':
+            extra_metrics.append('npullups')
+
+        if self.round.tournament.pref('draw_pullup_restriction') == 'easy_draw':
+            extra_metrics.append('draw_strength_score')
```

Comment:
Also, if either of these two metrics are _in_ the team standings precedence, then the draw fails to generate for adding a metric twice.

I'm open to dropping this restriction on standings generation though, I can't quite remember why it's there other than to catch user error quickly. (Some metrics mean something different if repeated, namely who-beat-whom.)